css-pseudo-4/Overview.bs

```diff
@@ -1337,6 +1337,43 @@ details[open]::details-content {
 }</pre>
   </div>
 
+<h3 id="picker-pseudo">
+Styling form control pickers: the ''::picker()'' pseudo-element</h3>
+
+  The <dfn>::picker()</dfn> pseudo-element targets the popup picker of form
+  control elements which have popup pickers, such as the <{select}> element. It
+  is a [=part-like pseudo-element=].
+
+  <pre class=prod>
+    ::picker() == ::picker( <<ident>>+ )
+  </pre>
+
+  The ''::picker()'' pseudo-element only matches when the <a>originating
+  element</a> supports <a>base appearance</a> and has a popup picker. The
+  specified <<ident>> must also match the unique picker name of the
+  <a>originating element</a>. For example, the unique picker name for the
+  <{select}> element is "select".
+
+  There is no restriction on which properties apply to the ''::picker()''
+  pseudo-element.
+
+  In order for the ''::picker()'' pseudo-element to be rendered, it and its
+  <a>originating element</a> must be rendered with <a>base appearance</a> by
+  applying ''appearance: base'' to both of them.
+
+  <pre class="example">
+&lt;style>
+  select, select::picker(select) {
+    appearance: base;
+  }
+  select::picker(select) {
+    border: 5px solid red;
+    background-color: blue;
+  }
+&lt;/style>
+  </pre>
+
```

Comment:
I think it's probably worth adding a note (you can just start a paragraph with "Note: " and it will get the right styles) that once all of the controls that might support picker styling have it, then we might add a (no-argument) `::picker` pseudo-element as well.  (I think it's worth adding because this may be a ways in the future, and the people who made these plans today might not be around to do that part!)

Reply:
I agree. @fantasai are you ok with this?

Reply:
I think you can go ahead without an explicit response from @fantasai here given that it matches the last sentence of the middle bullet of https://github.com/w3c/csswg-drafts/issues/10440#issuecomment-2303554166 .